include/odp/api/spec/traffic_mngr.h

```diff
@@ -189,6 +189,47 @@ extern "C" {
  * tree/hierarchy of nodes.
  */
 
+/**
+ * TM node stats capability
```

Comment:
To make the statistics capabilities documentation uniform:
`TM node statistics capabilities`